tests/end-to-end/director/labeling_api_test.go

```diff
@@ -563,3 +565,173 @@ func TestDeleteDefaultValueInScenariosLabelDefinition(t *testing.T) {
 	require.Error(t, err)
 	assert.EqualError(t, err, errMsg)
 }
+
+func TestSearchApplicationsByLabels(t *testing.T) {
+	// GIVEN
+	//TODO: enable this test after implementing filtering on applications
+	t.SkipNow()
```

Comment:
Please edit description of the task "DB for Applications domain". We need to remember to unskip it.